ledger/test-common/src/main/daml/semantic/DivulgenceTests.daml

```diff
@@ -0,0 +1,61 @@
+-- Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module DivulgenceTests where
+
+template Contract
+  with
+    party: Party
+  where
+    signatory party
+
+template Divulgence
+  with
+    divulgee: Party
+    divulger: Party
+  where
+    signatory divulgee, divulger
+    controller divulger can
+      nonconsuming Divulge: ()
+        with contractId: ContractId Contract
+          -- create an event so the transaction is visible on the GetTransactionTrees endpoint
+          do create Dummy with holder=divulger
+             fetch contractId
+             return ()
+      nonconsuming CreateAndDisclose: ContractId Contract
+          do create Contract with party=divulger
+    controller divulgee can
+      nonconsuming CanFetch: ContractId Dummy
+        with contractId: ContractId Contract
+          do fetch contractId
+             -- create an event so the transaction is visible on the GetTransactionTrees endpoint
+             create Dummy with holder=divulgee
+
+template DivulgenceHelper
```

Comment:
Consider renaming to `DivulgenceProposal` and replacing `Sign` with `Accept`.

Reply:
Done